assets/js/util.js

```diff
@@ -257,6 +273,13 @@ function getURLParam(name) {
     return results === null ? '' : results[1];
 }
 
+// From: http://stackoverflow.com/a/19696443/1266600 (source: AOSP)
+function isURL(text) {
```

Comment:
ESLint can be disabled for this function. I think you/I added the enable afterwards but forgot the disable.